.github/workflows/pull_request.yml

```
@@ -4,6 +4,9 @@ name: CI
 on:  # yamllint disable-line rule:truthy
   pull_request:  # yamllint disable-line rule:empty-values
 
+env:
+  PYTHON_MATRIX: "3.8, 3.9, 3.10, 3.11"
```

Nit: it seems like the strings are quoted explicitly only in places where there's ambiguous floats. All other places don't add quotes unnecessarily, which happens to be what I prefer as well. How about making this more consistent with the style around?
```suggestion
  PYTHON_MATRIX: 3.8, 3.9, 3.10, 3.11
```